atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -199,21 +209,49 @@ private SweepResults doRun(TableReference tableRef,
     }
 
     /**
-     * Returns the number of deleted (cell, timestamp) pairs.
+     * Returns the number of blocks that were deleted.
```

Comment:
Yea `block` is new and vague to me? Versions of a cell or cell versions would have made more sense to me given previous context.